contrib/internal/fasthttptrace/fasthttptrace.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package fasthttptrace
+
+import (
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal"
+
+	"github.com/valyala/fasthttp"
+)
+
+// StartSpanFromFastHTTPContext returns a new span with the given operation name and options.
```

Comment:
```suggestion
// StartSpanFromContext returns a new span with the given operation name and options.
```

Reply:
Maybe we could also clarify in this comment that this does the same as `tracer.StartSpanFromContext` but using a fasthttp context.